Shared/Strings/Strings.swift

```diff
@@ -140,6 +140,8 @@ internal enum L10n {
   internal static let ascending = L10n.tr("Localizable", "ascending", fallback: "Ascending")
   /// Aspect Fill
   internal static let aspectFill = L10n.tr("Localizable", "aspectFill", fallback: "Aspect Fill")
+  /// Only EAC-3/DD+ Atmos tracks will play back with spatial audio. TrueHD Atmos will be transcoded to multichannel audio. Requires Jellyfin ffmpeg 7.1.1-5+
+  internal static let atmosCompatibilityDescription = L10n.tr("Localizable", "atmosCompatibilityDescription", fallback: "Only EAC-3/DD+ Atmos tracks will play back with spatial audio. TrueHD Atmos will be transcoded to multichannel audio. Requires Jellyfin ffmpeg 7.1.1-5+")
```

Comment:
Please do not explicitly add translations in PRs. 

> Wait, that's intentional - I added translations for all languages. I rebased today. Per the [contribution guide](https://github.com/jellyfin/Swiftfin/blob/main/Documentation/contributing.md): "new strings that are not part of an experimental feature must be localized"

We could be a bit more clear by rephrasing: "new strings that are not part of an experimental feature must be added for localization". We use Weblate for our localizations.